pipelines/README.md

```diff
@@ -44,6 +44,19 @@ After creating the S3 bucket and uploading the models  (see above), fill out `aw
 oc create -f tekton/azureml-container-pipeline/aws-env.yaml
 ```
 
+### Provide Git credentials
+
+See the GitOps pipeline section below for more details on how to do this, e.g.:
+
+```bash
+# Bike rentals app
+cp pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml /tmp/gitea-edge-user-1-secret.yaml
```

Comment:
The above instructions assume that we are in the `pipelines` directory and drop that leading `pipelines/` path segment, we should be consistent here (or add it there).

Also, is there a specific reason why not just edit the `example-git-credentials-secret.yaml` in-place and `oc apply` that, like we did with `aws-env.yaml` above?